packages/api/src/utils/psa.js

```diff
@@ -132,11 +149,25 @@ export function transformAndValidate (payload) {
   // Validate CID.
   if (cid) {
     try {
-      normalizedCid = normalizeCid(cid)
-      opts.cid = cid
+      let c
+      try {
+        c = CID.parse(cid)
+        normalizedCid = c.toV1().toString()
+        opts.cid = cid
+      } catch (e) {
+        throw new PSAErrorInvalidData(INVALID_CID)
+      }
+
+      if (!SUPPORTED_CODECS.includes(c.code)) {
+        let message = PSACodecNotSupported.MESSAGE
+        if (c.code === 72) {
+          message = `${message} If you're trying to pin using an IPNS record that isn't supported yet. Please +1 [this githib issue](https://github.com/web3-storage/web3.storage/issues/2155) if you want it to be.`
```

Comment:
```suggestion
          message = `${message} If you're trying to pin using an IPNS record that isn't supported yet. Please +1 [this github issue](https://github.com/web3-storage/web3.storage/issues/2155) if you want it to be.`
```
Does the client parsing this error message support markdown syntax for hyperlinks?

Reply:
Good point, while obv this API can be used directly, I think it's safe to expect most of the requests would probably come from Kubo. And I don't know if it does support it.
I wonder if it's a real problem thogh. Even if it doesn't, Markdown syntax for links it's quite common anyway, and I expect users to be able to still understand and navigate to the issue.

What do you think?